src/locales/en.json5

```diff
@@ -126,6 +126,7 @@
         "create": "@:audit.action.default.create",
         "update": "@:audit.action.default.update",
         "update_draft_set": "Create or Update Draft",
+        "update_draft_replace": "Replace Draft",
```

Comment:
I think it'd be good to add this action to the filter in `AuditFiltersAction`. We can't add the upgrade event as easily, since Backend doesn't know how to filter for `upgrade.*` events as a category. However, since it's easy to add this event, and it will be returned as part of the general "Form Actions" category (/v1/audits?action=form), I think we should add `form.update.draft.replace` to the filter. No test needed, just adding it to the list.